native/cocos/2d/renderer/UIMeshBuffer.cpp

```diff
@@ -172,6 +172,11 @@ void UIMeshBuffer::parseLayout() {
     _meshBufferLayout = reinterpret_cast<MeshBufferLayout*>(_sharedBuffer);
 }
 
+// only initialize needCreateLayout = true call it
+void UIMeshBuffer::freeLayout() {
```

Comment:
Should be public? It seems is used internally.

Reply:
And can free it in destructor?

Reply:
done